src/views/product-integrations-landing/components/loading-skeleton/integrations-loading-skeleton.module.css

```
@@ -0,0 +1,76 @@
+.root {
+	margin-top: 32px;
+}
+
+.placeholder {
+	--loading-skeleton-animation-duration: 4s;
+
+	animation: loading var(--loading-skeleton-animation-duration) infinite;
+	background-color: var(--token-color-palette-alpha-100);
+	background-image: linear-gradient(
+		90deg,
+		transparent,
+		var(--token-color-palette-alpha-100),
+		var(--token-color-palette-alpha-200),
+		transparent
+	);
+	background-position: -300% 0;
+	background-repeat: no-repeat;
+	background-size: 75%;
+	border-radius: 2px;
+	opacity: 0.55;
+	width: 100%;
+
+	/* Only enable faster animation if query is supported and value is no-preference */
+	@media (prefers-reduced-motion: no-preference) {
+		--loading-skeleton-animation-duration: 2s;
+	}
+}
```

Slowing both animation durations by `0.5s` after chatting with @braydenlove and @kellyschummer in Slack. 💖

```suggestion
.placeholder {
	--loading-skeleton-animation-duration: 4.5s;

	animation: loading var(--loading-skeleton-animation-duration) infinite;
	background-color: var(--token-color-palette-alpha-100);
	background-image: linear-gradient(
		90deg,
		transparent,
		var(--token-color-palette-alpha-100),
		var(--token-color-palette-alpha-200),
		transparent
	);
	background-position: -300% 0;
	background-repeat: no-repeat;
	background-size: 75%;
	border-radius: 2px;
	opacity: 0.55;
	width: 100%;

	/* Only enable faster animation if query is supported and value is no-preference */
	@media (prefers-reduced-motion: no-preference) {
		--loading-skeleton-animation-duration: 2.5s;
	}
}
```